### ISSUE TITLE
Add ESBuild + Rollup benchmark

### ISSUE THREAD

**Issue by ImedAdel** [evanw/esbuild#318]
TLDR: ESBuild as a minifier (instead of Terser) for Rollup is twice as fast as Rollup with Terser

I needed a plugins API along with ESBuild so I benchmarked it along with Rollup and I found that it can run twice as fast than Rollup with Terser. Specifically, I added the following lines to the `Makefile`. I guess adding it to the official benchmark would be convincing for some people to switch from Terser to ESBuild?

```makefile
THREE_ROLLUP_CONFIG += import { startService } from 'esbuild'; let service; export default { output: { format: 'iife', name: 'THREE', sourcemap: true }, plugins: [{ name: 'esbuild', async buildStart() { if (!service) { service = await startService() } }, async renderChunk(code, chunk) { if(service) { const result = await service.transform(code, { target: 'es2020', sourcefile: chunk, minify: true, sourcemap: true }); return { code: result.js, map: result.jsSourceMap }; } return { code: '', map: undefined }; }, generateBundle() { if (service) { service.stop(); service = undefined } }, }], }
```

|                   |  real |  usr  |  sys |
|-------------------|-----:|-----:|----:|
| ESBuild           |  1.35 |  1.94 | 0.47 |
| Rollup w/ ESBuild | 24.89 | 32.45 | 1.81 |
| Rollup w/ Terser  | 68.28 | 85.79 | 3.21 |

PS. I just realized that there is a plugins branch. It's still not clear what to return (both in JS and Go), but I guess I'll try it instead of relying on Rollup.

**Comment by evanw** [comment]
Closing. I'm not currently planning to do this. There's a lot of different combinations of everything that could be added to the benchmark but I feel like that's visually overwhelming and makes it harder to read.
